build/root/Makefile.generated_files

```diff
@@ -659,6 +660,10 @@ gen_conversion: $(CONVERSION_FILES) $(CONVERSION_GEN)
 verify_gen_conversion: $(CONVERSION_GEN)
 	$(RUN_GEN_CONVERSION) --verify-only
 
+.PHONY: verify_gen_api_resource_list
+verify_gen_api_resource_list:
```

Comment:
is this verification only? I.e. there is no code generation involved? Then this probably belongs into hack/verify-all.sh

Reply:
Removed from generated_files make rules. The verification script is included in verify-all automatically (with verify- prefix)